libraries/standard/mqtt/src/private/iot_mqtt_helper.h

```diff
@@ -278,6 +288,6 @@ IotMqttError_t _IotMqtt_ProcessIncomingPublishFlags( uint8_t publishFlags,
  * @return Pointer to the end of the encoded string, which will be identical to
  * `pDestination` if nothing was encoded.
  */
- uint8_t * _IotMqtt_EncodeUserName( uint8_t * pDestination,
-                                  const IotMqttConnectInfo_t * pConnectInfo );
-#endif
\ No newline at end of file
+uint8_t * _IotMqtt_EncodeUserName( uint8_t * pDestination,
+                                   const IotMqttConnectInfo_t * pConnectInfo );
+#endif /* ifndef IOT_MQTT_SERIALIZE_INTERNAL_H_ */
```

Comment:
It looks like it was this way before the change, but is the header guard intentionally different from the file name?

Reply:
Good catch. It is cut and paste error. Will fix it.